way_to_home/user_profile/models.py

```diff
@@ -1 +1,53 @@
 """This module implements class that represents the user profile entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
```

Comment:
```suggestion
from user.models import User
from django.db import models, IntegrityError
from utils.abstract_models import AbstractModel
```

Optimize imports, please